src/commonMain/kotlin/assertk/assertions/string.kt

```diff
@@ -47,6 +48,26 @@ fun Assert<String>.contains(other: CharSequence, ignoreCase: Boolean = false) =
     expected("to contain:${show(other)} but was:${show(actual)}")
 }
 
+/**
+ * Asserts the string contains the expected strings.
+ * @param ignoreCase true to compare ignoring case, the default if false.
+ */
+fun Assert<String>.containsSubstrings(vararg expected: String, ignoreCase: Boolean = false) {
+    containsSubstrings(expected.toList(), ignoreCase)
+}
+
+/**
+ * Asserts the string contains the expected strings.
+ * @param ignoreCase true to compare ignoring case, the default if false.
+ */
+fun Assert<String>.containsSubstrings(expected: List<String>, ignoreCase: Boolean = false) {
+    assertAll {
+        expected.forEach {
+            contains(it, ignoreCase)
```

Comment:
this simply delegates to the already existing `Assert<String>.contains()`. another possibility would be to "inline" the logic like this:

```kotlin
fun Assert<String>.containsSubsrings(expected: List<String>, ignoreCase: Boolean = false) = given { actual ->
    expected.forEach {
        if (!actual.contains(it, ignoreCase)) {
            expected("to also contain:${show(it)} but was:${show(actual)}")
        }
    }
}
```

any ideas/opinions about that alternative?

Reply:
good point, error message. 

GIVEN:
```kotlin
assertThat("test").containsSubstrings("foo", "bar")
```

CURRENT OUTPUT:
```
The following assertions failed (2 failures)
	expected to contain:<"foo"> but was:<"test">
	expected to contain:<"bar"> but was:<"test">
```

changed/improved it to:
```
expected to contain:<["foo", "bar"]> but was:<"test">
```